src/main/java/com/hyperwallet/clientsdk/util/HyperwalletApiClient.java

```diff
@@ -132,6 +138,15 @@ protected void checkErrorResponse(final Response response) throws ParseException
         }
     }
 
+    private void checkResponseHeader(Response response) {
+        String contentTypeHeader = response.getHeader(CONTENT_TYPE_HEADER);
+        boolean isNotValidContentType = !contentTypeHeader.equals(VALID_JSON_CONTENT_TYPE) &&
+                !contentTypeHeader.equals(VALID_JSON_JOSE_CONTENT_TYPE);
+        if (isNotValidContentType) {
```

Comment:
can we also put the whole line 143 inside the if condition of line 145 the variable is not used elsewhere other than this if clause